server/server.go

```diff
@@ -519,12 +520,56 @@ func (server HTTPServer) singleRuleEndpoint(writer http.ResponseWriter, request
 		return
 	}
 
+	if rule.Internal == true {
```

Comment:
I think just `if rule.Internal {` is slightly better, but it's a nitpick again :)

Reply:
thanks:)